jvm-system/src/main/scala/com/avast/server/toolkit/system/SystemModule.scala

```diff
@@ -0,0 +1,62 @@
+package com.avast.server.toolkit.system
+
+import java.io.{OutputStream, Reader}
+import java.security.SecureRandom
+
+import cats.effect.{Clock, Sync}
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+import com.avast.server.toolkit.system.console.Console
+import com.avast.server.toolkit.system.random.Random
+
+import scala.language.higherKinds
+import scala.{Console => SConsole}
+
+/** Provides JVM system services such as standard in/out, time and random number generation.
+  *
+  * Contains already initialized standard [[com.avast.server.toolkit.system.console.Console]], default [[cats.effect.Clock]]
+  * and [[com.avast.server.toolkit.system.random.Random]] or you can use `make*` factory methods to create more or customized instances.
+  */
+class SystemModule[F[_]: Sync](val console: Console[F], val clock: Clock[F], val random: Random[F]) {
```

Comment:
Why is `Console` and `Random` tighten together? I would prefer to split them as they are not related to each other.

Reply:
Split. `ClockModule` does not make sense because creation of Cats `Clock` is very easy so I will just remove it.